pkg/injector/patch.go

```diff
@@ -36,7 +36,6 @@ func (wh *mutatingWebhook) createPatch(pod *corev1.Pod, req *v1beta1.AdmissionRe
 		WithLabelValues().Observe(elapsed.Seconds())
 	originalHealthProbes := rewriteHealthProbes(pod)
 
-	wh.meshCatalog.ExpectProxy(cn)
```

Comment:
I think we could also clean up the ExpectProxy now that the injector cannot really tell osm any of it.

Reply:
Open issue: https://github.com/openservicemesh/osm/issues/2481